sdk/sdk.yaml

```diff
@@ -1664,127 +1706,127 @@ paths:
   #                   type: boolean
   #                   description: 'Whether the given wallet owns any token in the given NFT collection.'
   #     operationId: sdk-isHolderOfCollection
-  
+
   # ////////////////////////////////////////////////////////////////////////////////
   # /////////////////////////////////// TRANSACT ENDPOINTS /////////////////////////
   # ////////////////////////////////////////////////////////////////////////////////
 
   /{apiKey}/getTransaction:
-      post:
-        operationId: sdk-getTransaction
-        summary: getTransaction
-        description: 'Returns the information about a transaction requested by transaction hash or number. In the response object, `blockHash`, `blockNumber`, and `transactionIndex` are `null` when the transaction is pending.'
-        tags: ['SDK Transact Endpoints']
-        parameters:
-          - name: apiKey
-            in: path
+    post:
+      operationId: sdk-getTransaction
+      summary: getTransaction
+      description: 'Returns the information about a transaction requested by transaction hash or number. In the response object, `blockHash`, `blockNumber`, and `transactionIndex` are `null` when the transaction is pending.'
+      tags: ['SDK Transact Endpoints']
+      parameters:
+        - name: apiKey
+          in: path
+          schema:
+            type: string
+            default: docs-demo
+            description: For higher throughput, **[create your own API key](https://alchemy.com/?a=docs-demo)**
+          required: true
+      requestBody:
+        content:
+          application/json:
             schema:
-              type: string
-              default: docs-demo
-              description: For higher throughput, **[create your own API key](https://alchemy.com/?a=docs-demo)**
-            required: true
-        requestBody:
+              $ref: sdk_body.yaml#/eth_getTransactionByHash
+              example:
+                params:
+                  [
+                    '0x88df016429689c079f3b2f6ad39fa052532c56795b733da78a91ebe6a713944b',
+                  ]
+      x-readme:
+        explorer-enabled: false
```

Comment:
@SahilAujla we should hide the explorer in verifyNftOwnership like we do for this method since the sample request wont actually work

Reply:
@elanh The requested changes are made ✅